app/services/change_school_responsible_body_service.rb

```diff
@@ -31,14 +34,26 @@ def log_error(e)
     Sentry.capture_exception(e)
   end
 
-  def update_preorder_information!
-    school.preorder_information.refresh_status!
+  def remove_school_from_current_pool
+    initial_responsible_body.remove_school_from_virtual_cap_pools!(school)
+  end
+
+  def school_can_be_added?
+    new_responsible_body.can_school_be_added_to_virtual_cap_pools?(school)
+  end
+
+  def school_can_be_removed?
+    initial_responsible_body&.can_school_be_removed_from_virtual_cap_pools?(school)
   end
 
-  def update_school!
+  def set_school_new_responsible_body
     school.update!(
-      responsible_body_id: new_responsible_body_id,
+      responsible_body_id: new_responsible_body.id,
       computacenter_change: COMPUTACENTER_CHANGE_STATUS,
     )
   end
+
+  def update_preorder_information
```

Comment:
Bang here too?

Reply:
Same comment as above. I don´t really mind to rename it but maybe we should not